PowerEditor/src/Parameters.cpp

```diff
@@ -742,7 +742,7 @@ generic_string ThemeSwitcher::getThemeFromXmlFileName(const TCHAR *xmlFullPath)
 }
 
 
-winVer getWindowsVersion()
+winVer NppParameters::getWindowsVersion()

```

Comment:
No, it can't be as it sets value for member _platForm in switch statement below.

```C++
switch (si.wProcessorArchitecture)
	{
	case PROCESSOR_ARCHITECTURE_IA64:
		_platForm = PF_IA64;
		break;